crates/solidity/outputs/cargo/crate/src/lib.rs

```diff
@@ -9,11 +9,3 @@ pub use generated::*;
 mod supress_cli_dependencies {
     use {anyhow as _, clap as _, serde_json as _};
 }
-
```

Comment:
I removed this from the Rust crate and moved the check to the node addon crate's build dependency to be clear that it's only relevant there (and seeing a dependency on oneself looks confusing at first)